java-bootcamp-2018/src/com/globant/abstractFactory/MicrosoftSQLFactory.java

```diff
@@ -0,0 +1,26 @@
+package com.globant.abstractFactory;
+
+public class MicrosoftSQLFactory implements ISQLConnection {
+	static final String MICROSOFTACCESS = "MicrosoftAccess";
+	static final String MICROSOFTSQLSERVER = "MicrosoftSQLServer";
+
+	@Override
+	public IMicrosoftSQL getMicrosoftSQL(String sql) {
+		if (sql == null) {
+			return null;
+		}
+		if (MICROSOFTACCESS.equalsIgnoreCase("sql")) {

```

Comment:
This condition will always be false. Please refactor to do the right validation.